bff/samples/Apis/Api.DPoP/Extensions.cs

```diff
@@ -33,7 +34,7 @@ public static WebApplication ConfigureServices(this WebApplicationBuilder builde
             });
 
         // layers DPoP onto the "token" scheme above
-        services.ConfigureDPoPTokensForScheme("token");
+        Duende.AspNetCore.Authentication.JwtBearer.DPoP.DPoPServiceCollectionExtensions.ConfigureDPoPTokensForScheme(services, "token");
```

Comment:
Could this just be a using?

Reply:
Looks like the call to an extension method was changed to it's static invocation form.